packages/gestalt/src/icons/megaphone.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><path fill="#B5B5B5" d="M14,17.499858 C14,18.877858 12.878,19.999858 11.5,19.999858 C10.122,19.999858 9,18.877858 9,17.499858 L9,16.999858 L14,16.999858 L14,17.499858 Z M22.947,15.629858 L18.701,2.71185796 C18.483,2.04985796 17.697,1.78885796 17.135,2.19085796 L1.435,13.430858 C1.065,13.694858 0.91,14.174858 1.053,14.609858 L1.604,16.287858 C1.744,16.712858 2.136,16.999858 2.578,16.999858 L7,16.999858 L7,17.499858 C7,19.984858 9.015,21.999858 11.5,21.999858 C13.985,21.999858 16,19.984858 16,17.499858 L16,16.999858 L21.973,16.999858 C22.673,16.999858 23.168,16.303858 22.947,15.629858 Z"/></svg>
```

Comment:
@akleiner2 could you ask @jaymarsh to run this through the SVGO optimizer?

![image](https://user-images.githubusercontent.com/127199/42789858-7751f5c0-891c-11e8-9c74-77b02e10932e.png)